Documentation/PropertyWrapper/Parameter.md

```diff
@@ -280,10 +280,10 @@ Exposing the `Component` on a web socket interface requires the client to send t
 }
 ```
 
-## Parameters Defining An Endpoint
+## Path Parameters
 
 Some middleware types and protocols can expose parameters as part of the endpoint defining charactersistics. E.g. RESTful and OpenAPI based APIs use the URI path to define endpoints using variables such as `/birds/BIRD_ID` to identify the `Bird` the request is target at using the `BIRD_ID`.
```

Comment:
_targeted_  or more easy: "identify the requested `Bird` by using `BIRD_ID` as identifier"